action.yml

```diff
@@ -8,6 +8,10 @@ inputs:
   github-token:
     description: 'Basically GITHUB_TOKEN'
     required: true
+  jobs-to-wait-for:
+    description: 'Comma-separated list of job names to wait for'
```

Comment:
As I understand it, job names might be duplicated if they are defined in another workflow. So I want to specify combined `workflow file name` and `job name` when providing this feature.

I guess https://github.com/kachick/wait-other-jobs/pull/475#issuecomment-1539723269 is a way to get them.